DfE.FindInformationAcademiesTrusts/Services/ExportService.cs

```diff
@@ -0,0 +1,99 @@
+﻿using ClosedXML.Excel;
+using DfE.FindInformationAcademiesTrusts.Data;
+
+namespace DfE.FindInformationAcademiesTrusts.Services.Export
+{
+    public interface IExportService
+    {
+        byte[] ExportAcademiesToSpreadsheetUsingProvider(Data.Trust? trust, Trust.TrustSummaryServiceModel? trustSummary);
+    }
+
+    public class ExportService : IExportService
+    {
+
+        public byte[] ExportAcademiesToSpreadsheetUsingProvider(Data.Trust? trust, Trust.TrustSummaryServiceModel? trustSummary)
+        {
+            var headers = new List<string>
+            {
+                "School Name", "URN", "Local Authority", "Type", "Rural or Urban", "Date joined",
+                "Previous Ofsted Rating", "Before/After Joining","Date of Previous Ofsted",
+                "Current Ofsted Rating", "Before/After Joining", "Date of Current Ofsted",
+                "Phase of Education", "Pupil Numbers", "Capacity", "% Full", "Pupils eligible for Free School Meals"
+            };
+
+            var academies = trust?.Academies ?? [];
+
+            var dataExtractor = new Func<Data.Academy, object[]>((academy) =>
+            [
+                academy.EstablishmentName ?? string.Empty,
+                academy.Urn,
+                academy.LocalAuthority ?? string.Empty,
+                academy.TypeOfEstablishment ?? string.Empty,
+                academy.UrbanRural ?? string.Empty,
+                academy.DateAcademyJoinedTrust.ToString("yyyy-MM-dd"),
```

Comment:
Instead of repeating the "yyyy-MM-dd" it should be added as to StringFormatConstants, possibility as "ExportDate" or something similar

Reply:
Great shout, thank you, done